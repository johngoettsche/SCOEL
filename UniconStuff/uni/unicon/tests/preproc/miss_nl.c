/* no trailing new-line */
0