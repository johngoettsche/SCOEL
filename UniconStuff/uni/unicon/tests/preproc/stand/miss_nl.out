#line 1 "miss_nl.c"

0

